projects/github.com/bcicen/ctop/package.yml

```diff
@@ -0,0 +1,31 @@
+distributable:
+  url: https://github.com/bcicen/ctop/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+display-name: ctop
+
+versions:
+  github: bcicen/ctop
+
+build:
+  dependencies:
+    go.dev: ^1.18
+    git-scm.org: "*"
+
+  script: |
+    go mod download
+    CGO_ENABLED="$CGO_ENABLED" go build -tags release -ldflags="$GO_LDFLAGS" -o "$BUILDLOC"
+  env:
+    CGO_ENABLED: 0
+    BUILDLOC: "{{prefix}}/bin/ctop"
+    BUILD: "$(git rev-parse --short HEAD)"
```

Comment:
Technically, this isn't _really_ right, since it'll be the non commit in the fake git checkout. Better to just set it to `pkgx`